Content.Client/Actions/ActionsSystem.cs

```diff
@@ -72,7 +71,8 @@ private void HandleComponentState(EntityUid uid, ActionsComponent component, ref
                     continue;
                 }
 
-                act.CopyFrom(serverAct);
+                component.Actions.Remove(act);
+                component.Actions.Add(_serialization.Copy(serverAct));
```

Comment:
Because of actions shitcode its possible that this might need to be `_serialization.Copy(serverAct, ref act);` instead (i.e., copy into the existing action instead of creating a new one).

Its also possible the issue this was causing was since fixed? I don't really remember, but probably better to keep as is if possible. Eventually actions will need to be re-refactored anyways to try fix all the jank,